pages/reset/index.js

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from 'react'
+import Router from 'next/router'
+
+const Index = () => {
+  useEffect(() => {
+    Router.push('/reset/step1')
```

Comment:
I didn't think about it too much. I implemented first idea which came to my mind. It can be anything. I would personally go for `dashboard.core.ac.uk/reset?verify={token}`...

Reply:
Both are fine to me. It depends on how you are going to handle stuff: in a single file or using 2 ones.

Reply:
dashboard.core.ac.uk/reset/{token} it is then.

@valerapro please can you update reset link to the following URL:

 dashboard.core.ac.uk/reset/{token}?email="email@here.com"

Reply:
You should send the link along with the rest request. The token is fulfilled by the placeholder replacement and inserted to email.